apps/fluent-tester/src/testPages.ts

```diff
@@ -78,13 +78,13 @@ export const tests: TestDescription[] = [
     platforms: ['android', 'macos', 'win32'],
   },
   {
-    name: 'Checkbox',
```

Comment:
Yeah I'm in the process of trying to combine Checkbox,

For Link and Radiogroup, two pages is expected as they're under construction still.
The rest of the components we would need to clean up.

The main difficulty with combining the test pages is getting E2E testing to cooperate, since there's two controls on one page. That was an issue I ran into when trying to combine checkbox last time.